client.go

```diff
@@ -26,6 +26,8 @@ type Client struct {
 //
 // If the addr includes a scheme, it must be "amqp" or "amqps".
 // If no port is provided, 5672 will be used for "amqp" and 5671 for "amqps".
+// If username and password information is not empty it's used as SASL PLAIN
```

Comment:
Can you add an empty comment line between these sentences? Otherwise godoc sticks them together as a single paragraph.

Reply:
done